tally_ho/apps/tally/migrations/0028_resultformstats_has_de_error.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.1.1 on 2020-03-23 18:37
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('tally', '0027_auto_20200320_1933'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='resultformstats',
+            name='has_de_error',
```

Comment:
let's name this explicitly as `data_entry_errors`, we can also make it a count with default 0

Reply:
@pld sure we can do that but i'll suggest, we name it `data_entry_error` instead, with a default of 0 or False because it's value can either be `true or false`.

Reply:
cool, but I'm suggesting we change this so that we store a count of the number of errors instead of a boolean here

Reply:
Cool, got you

Reply:
@pld if we count the `number of errors` and they happen to be more than the forms submitted, are we okay with having this as more than 100%

Reply:
no, this shouldn't be expressed as a percentage

Reply:
@pld when you suggested we rename `has_de_error` to `data_entry_errors` this meant that we will be recording the number of errors caused by a DE1 or DE2 user during corrections here. No?

If yes, this will mean that the percentage of errors will be the summation of all the errors caused by that clerk using `data_entry_errors` field and divide by the total forms processed.

NB: Keep in mind the `data_entry_errors` field is associated with a single result form in the `result_form_stat` object.

Reply:
This is what will lead to the percentage being over 100% which is not idle.

Reply:
This pr https://github.com/onaio/tally-ho/pull/268 has logic that uses `data_entry_errors` field, check this line https://github.com/onaio/tally-ho/blob/cbdbddc73f18fe938cc552442bf1e4519460eb00/tally_ho/apps/tally/views/reports/staff_performance_metrics.py#L176

Reply:
I guess what I'm saying is that I'm not sure what the total number of errors divided by the total forms processed would give you, but it wouldn't be the percentage of errors.

To get the percentage of forms with errors we can do per user the count(data_entry_errors > 0) / total number of forms. With the extra data we could also come up with some other metrics later, but let's not worry about this now.

The reason I'm pushing for this being a count is so we don't lose the information, in case it turns out to be valuable later on.

Reply:
Nice!